.github/workflows/main.yml

```diff
@@ -0,0 +1,65 @@
+name: Deploying an app
+
+on:
+  workflow_dispatch:
+  push:
+    branches: ["dev", "main"]
```

Comment:
why are we deploying app both on push and on pull request for main?

Reply:
changed it to updates rather than main